disnake/user.py

```diff
@@ -111,6 +116,9 @@ def _update(self, data: Union[UserPayload, PartialUserPayload]) -> None:
         self._public_flags = data.get("public_flags", 0)
         self.bot = data.get("bot", False)
         self.system = data.get("system", False)
+        self.collectibles: Optional[Collectibles] = None
+        if collectibles_data := data.get("collectibles"):
+            self.collectibles = Collectibles(self._state, collectibles_data)
```

Comment:
I'd consider lazily initializing this in a `@property`, similar to `_avatar_decoration_data`, since users can be received often and in many different ways. Only rarely is someone interested in the collectibles, so parsing this every time would be a waste of resources.

Reply:
Adding on to this, what about returning an empty `Collectibles` object instead of `None` when there are no collectibles? Otherwise people would have to check `user.collectibles is not None and user.collectibles.nameplate is not None`.

Reply:
> Adding on to this, what about returning an empty `Collectibles` object instead of `None` when there are no collectibles? Otherwise people would have to check `user.collectibles is not None and user.collectibles.nameplate is not None`.

https://github.com/DisnakeDev/disnake/pull/1303/commits/b702fb17a0842500b99f29d1ff033ad75a6c4342